workstation setup instructions are broken for Fedora

I am unable to build any native_posix packages due to missing C++ compiler, 32-bit libraries, and possibly other packages.

The workstation setup instructions have to be amended.

```
Compiling the C compiler identification source file "CMakeCCompilerId.c" failed.
Compiler: /usr/bin/gcc 
Build flags: -m32
Id flags:  

The output was:
1
/usr/bin/ld: cannot find crt1.o: No such file or directory
/usr/bin/ld: cannot find crti.o: No such file or directory
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-redhat-linux/7/libgcc_s.so when searching for -lgcc_s
/usr/bin/ld: cannot find -lgcc_s
/usr/bin/ld: cannot find -lc
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-redhat-linux/7/libgcc_s.so when searching for -lgcc_s
/usr/bin/ld: cannot find -lgcc_s
/usr/bin/ld: cannot find crtn.o: No such file or directory
collect2: error: ld returned 1 exit status


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: -m32
Id flags:  

The output was:
1
gcc: error trying to exec 'cc1plus': execvp: No such file or directory


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: -m32
Id flags: -c 

The output was:
1
gcc: error trying to exec 'cc1plus': execvp: No such file or directory


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: -m32
Id flags: --c++ 

The output was:
1
gcc: error: unrecognized command line option ‘--c++’


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: -m32
Id flags: --ec++ 

The output was:
1
gcc: error: unrecognized command line option ‘--ec++’; did you mean ‘-Weffc++’?


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: 
Id flags:  

The output was:
1
gcc: error trying to exec 'cc1plus': execvp: No such file or directory


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: 
Id flags: -c 

The output was:
1
gcc: error trying to exec 'cc1plus': execvp: No such file or directory


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: 
Id flags: --c++ 

The output was:
1
gcc: error: unrecognized command line option ‘--c++’


Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /usr/bin/gcc 
Build flags: 
Id flags: --ec++ 

The output was:
1
gcc: error: unrecognized command line option ‘--ec++’; did you mean ‘-Weffc++’?


Checking whether the CXX compiler is IAR using "" did not match "IAR .+ Compiler":
gcc: fatal error: no input files
compilation terminated.
Checking whether the CXX compiler is IAR using "" did not match "IAR .+ Compiler":
gcc: fatal error: no input files
compilation terminated.
Performing C SOURCE FILE Test toolchain_is_ok failed with the following output:
Change Dir: /projects/zephyr/samples/philosophers/out/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/gmake" "cmTC_e3e63/fast"
/usr/bin/gmake -f CMakeFiles/cmTC_e3e63.dir/build.make CMakeFiles/cmTC_e3e63.dir/build
gmake[1]: Entering directory '/projects/zephyr/samples/philosophers/out/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_e3e63.dir/src.c.obj
/usr/bin/gcc   -m32 -Dtoolchain_is_ok   -o CMakeFiles/cmTC_e3e63.dir/src.c.obj   -c /projects/zephyr/samples/philosophers/out/CMakeFiles/CMakeTmp/src.c
Linking C executable cmTC_e3e63
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_e3e63.dir/link.txt --verbose=1
/usr/bin/gcc -m32 -Dtoolchain_is_ok    CMakeFiles/cmTC_e3e63.dir/src.c.obj  -o cmTC_e3e63 
/usr/bin/ld: cannot find crt1.o: No such file or directory
/usr/bin/ld: cannot find crti.o: No such file or directory
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-redhat-linux/7/libgcc_s.so when searching for -lgcc_s
/usr/bin/ld: cannot find -lgcc_s
/usr/bin/ld: cannot find -lc
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-redhat-linux/7/libgcc_s.so when searching for -lgcc_s
/usr/bin/ld: cannot find -lgcc_s
/usr/bin/ld: cannot find crtn.o: No such file or directory
collect2: error: ld returned 1 exit status
gmake[1]: *** [CMakeFiles/cmTC_e3e63.dir/build.make:98: cmTC_e3e63] Error 1
gmake[1]: Leaving directory '/projects/zephyr/samples/philosophers/out/CMakeFiles/CMakeTmp'
gmake: *** [Makefile:126: cmTC_e3e63/fast] Error 2

Source file was:
int main(void) { return 0; }
```

--- a/doc/getting_started/installation_linux.rst
+++ b/doc/getting_started/installation_linux.rst
@@ -54,16 +54,19 @@ Install the required packages in a Ubuntu host system with:
 
    sudo apt-get install --no-install-recommends git cmake ninja-build gperf \
      ccache doxygen dfu-util device-tree-compiler \
-     python3-ply python3-pip python3-setuptools xz-utils file make gcc-multilib
+     python3-ply python3-pip python3-setuptools xz-utils file make gcc-multilib \
+     autoconf automake libtool
 
 Install the required packages in a Fedora host system with:
 
 .. code-block:: console
 
-   sudo dnf group install "Development Tools"
+   sudo dnf group install "Development Tools" "C Development Tools and Libraries"
    sudo dnf install git cmake ninja-build gperf ccache\
 	 doxygen dfu-util dtc python3-pip \
-	 python3-ply python3-yaml dfu-util dtc python3-pykwalify
+	 python3-ply python3-yaml dfu-util dtc python3-pykwalify \
+         glibc-devel.i686 libstdc++-devel.i686 autoconf automake libtool \
+         gcc-c++.i686
 
 Install additional packages required for development with Zephyr::
 

